modules/apps/transfer/client/cli/query.go

```diff
@@ -167,3 +167,36 @@ func GetCmdQueryDenomHash() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdQueryDenomHash defines the command to query a denomination hash from a given trace.
+func GetCmdQueryIBCTokenOut() *cobra.Command {
```

Comment:
Can we rename this (and other similar names) to something more explicit? Maybe `QueryTotalEscrowForDenom`?